<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Setting</class>
 <widget class="QWidget" name="Setting">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QPushButton" name="Page2">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>70</y>
     <width>100</width>
     <height>71</height>
    </rect>
   </property>
   <property name="toolTip">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;湿度&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <property name="text">
    <string>坐标轴设置</string>
   </property>
  </widget>
  <widget class="QPushButton" name="Page1">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>100</width>
     <height>71</height>
    </rect>
   </property>
   <property name="toolTip">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;温度&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
   <property name="text">
    <string>阈值设置</string>
   </property>
  </widget>
  <widget class="QStackedWidget" name="stackedWidget">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>0</y>
     <width>700</width>
     <height>400</height>
    </rect>
   </property>
   <property name="currentIndex">
    <number>0</number>
   </property>
   <widget class="Threshold" name="page_3"/>
   <widget class="Range" name="page_4"/>
  </widget>
 </widget>
 <customwidgets>
  <customwidget>
   <class>Threshold</class>
   <extends>QWidget</extends>
   <header location="global">threshold.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>Range</class>
   <extends>QWidget</extends>
   <header location="global">range.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
